code/obj/cement.dm

```diff
@@ -68,7 +61,7 @@
 	flags = FPRINT | CONDUCT | USEDELAY
 	var/strength = 0 // 1=poor, 2=ok, 3=good, 4=perfect
 	var/health = 30 //health num modified in New, 30 for poor, 60 for ok, 90 for good, 120 for perfect
```

Comment:
doesn't this var already exist on all objects? _health or something?

Reply:
Yep, refactored to use it now.